mail_notification_custom_subject/models/mail_message_custom_subject.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MailMessageCustomSubject(models.Model):
+    _name = "mail.message.custom.subject"
+    _description = "Mail Message Custom Subject"
+
+    name = fields.Char(string="Template Name")
```

Comment:
Does the name field have a meaning here? If not:

```suggestion
    _description = "Mail Message Custom Subject"
    _rec_name = "subject_template"
    _order = "subject_template"

```

Reply:
I think that's necessary for identify the message template. What do you think @joao-p-marques ?

Reply:
It is not strictly necessary, but it can help the UX IMHO, as it helps to identify each template more easily.